lib/features/login/data/network/config/authorization_interceptors.dart

```diff
@@ -77,8 +77,7 @@ class AuthorizationInterceptors extends InterceptorsWrapper {
   void onError(DioError err, ErrorInterceptorHandler handler) async {
     final requestOptions = err.requestOptions;
     log('AuthorizationInterceptors::onError(): $err');
-    if ((_isTokenExpired() || err.response?.statusCode == 401) &&
-        _isAuthenticationOidcValid()) {
+    if (_isTokenExpired() && _isAuthenticationOidcValid()) {
```

Comment:
Why we remove 401?

Reply:
There are many possible errors that return `statusCode=401`. We are currently dealing with `token` retrieval when `token expired`. So here we only need to care about the `token expired` condition to perform `refreshToken`. Avoid repeatedly calling refreshToken when some unexpected `401` error occurs.